test/test.dart

```diff
@@ -44,6 +44,11 @@ class _Person {
   late String name;
 }
 
+extension PersonJ on Person {
```

Comment:
do we really need this to be available to all tests. I suggest we move it to `app_test.dart` where it is used